### PR TITLE
upd: CI github action and docker images

### PR DESCRIPTION
**Pull request by itsyaasir**
# Issue ticket

Issue ticket link: <>

### Code conventions

- [x] I have performed a self-review of my code
- [x] Every new function is documented
- [x] Object names are auto explicative

### Security

- [x] The PR does not break APIs backward compatibility
- [x] The PR does not break the stable storage backward compatibility

### Testing

- [x] Every function is properly unit tested
- [x] I have added integration tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] IC endpoints are always tested through the `canister_call!` macro

### CODE REVIEW

File: .github/workflows/block-extractor-scheduler.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+              echo "rpc_url=testnet.bitfinity.network" >> $GITHUB_ENV
+          fi
+
+          if [ "${{ matrix.database }}" = "bigquery" ]; then
+              echo "db_project_id=${{ secrets.BIGQUERY_PROJECT_ID }}" >> $GITHUB_ENV
+              echo "db_dataset_id=${{ secrets.BIGQUERY_DATASET_ID }}" >> $GITHUB_ENV
+
+          elif [ "${{ matrix.database }}" = "postgres" ]; then
+              echo "db_username=${{ secrets.POSTGRES_USERNAME }}" >> $GITHUB_ENV
+              echo "db_password=${{ secrets.POSTGRES_PASSWORD }}" >> $GITHUB_ENV
```
Comment (ufoscout) [inline .github/workflows/block-extractor-scheduler.yml:46]: printing the password?!!?!? :0 
please remove it.
I would remove this entire set of prints anyway. The executables print the required information at startup

File: src/evm-block-extractor/Dockerfile [hunk 2]
```diff
@@ -16,6 +16,7 @@ RUN cargo build --release --bin evm-block-extractor-server
 FROM debian:bookworm-slim AS runtime
 
 ENV DATASET=testnet
+ENV DATABASE=bigquery
```
Comment (ufoscout) [inline src/evm-block-extractor/Dockerfile:19]: It should be `--bigquery`

Comment (ufoscout) [reply src/evm-block-extractor/Dockerfile:19]: or change it in the code to be used without `--`

**Review by ufoscout — CHANGES_REQUESTED** [review]

File: .github/workflows/block-extractor-scheduler.yml (hunk 1 above)
Comment (itsyaasir) [reply .github/workflows/block-extractor-scheduler.yml:46]: Not really! This command does not directly print the value of the secrets, GitHub secrets are encrypted environment variables created in a repository and can only be used in GitHub Actions workflows. The value is passed safely

File: src/evm-block-extractor/Dockerfile (hunk 2 above)
Comment (itsyaasir) [reply src/evm-block-extractor/Dockerfile:19]: If you check like 36; it is written like that.

File: .github/workflows/block-extractor-scheduler.yml (hunk 1 above)
Comment (itsyaasir) [reply .github/workflows/block-extractor-scheduler.yml]: Done

File: src/evm-block-extractor/Dockerfile (hunk 2 above)
Comment (itsyaasir) [reply src/evm-block-extractor/Dockerfile]: > I saw it, but line 36 is `--` `white_space` `bigquery` I don't know if ti works. Did you try it?

Yes; it does work

**Review by F3kilo — APPROVED** [review]

File: .github/workflows/block-extractor-scheduler.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+env:
+  RUST_LOG: info
+
+jobs:
+  backup-blocks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        database: [postgres, bigquery]
```
Comment (ufoscout) [inline .github/workflows/block-extractor-scheduler.yml:19]: Are we using postgres in gcp too?

File: .github/workflows/block-extractor-scheduler.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        run: mv target/release/evm-block-extractor evm-block-extractor
+
+      - name: Make binary executable
+        run: chmod +x evm-block-extractor
+
+      - name: Set Up Environment Variables
+        run: |
+          if [ "${{ matrix.network }}" = "mainnet" ]; then
+              echo "RPC_URL=mainnet.bitfinity.network" >> $GITHUB_ENV
+              echo "BIGQUERY_DATASET=BIGQUERY_DATASET_${{ matrix.network }}" >> $GITHUB_ENV
```
Comment (ufoscout) [inline .github/workflows/block-extractor-scheduler.yml:39]: this variable is required only if `matrix.database == 'bigquery'` so it should be in the `Run evm-block-extractor for BigQuery` block

**Review by ufoscout — CHANGES_REQUESTED** [review]

File: .github/workflows/block-extractor-scheduler.yml (hunk 3 above)
Comment (itsyaasir) [reply .github/workflows/block-extractor-scheduler.yml:19]: No

**Review by veeso — APPROVED** [review]

**Review by GraDKh — APPROVED** [review]

File: .github/workflows/block-extractor-scheduler.yml (hunk 3 above)
Comment (itsyaasir) [reply .github/workflows/block-extractor-scheduler.yml:19]: Are we not running the scheduler for postgres ? I thought we were running for both postgres and bigquery.

File: src/evm-block-extractor/Cargo.toml [hunk 5]
```diff
@@ -25,7 +25,7 @@ ethereum-json-rpc-client = { path = "../ethereum-json-rpc-client", features = [
 ] }
 serde = { workspace = true }
 serde_json = { workspace = true }
-sqlx = { workspace = true, features = ["postgres"] }
+sqlx = { workspace = true, features = ["postgres", "tls-native-tls"] }
```
Comment (ufoscout) [inline src/evm-block-extractor/Cargo.toml:28]: Use [rustls](https://crates.io/crates/rustls) instead

**Review by ufoscout — CHANGES_REQUESTED** [review]

**Review by GraDKh — APPROVED** [review]

**Review by ufoscout — APPROVED** [review]

**Pull request merged**